ultimate-pipeline/conf/base/catalog.yml

```diff
@@ -9,7 +9,7 @@ supervisely_video_annotations:
 
 yolo_annotation_txt: 
   type: partitions.PartitionedDataset
-  path: data/processed/mini_test_set/annotations/  # path to the location of partitions
+  path: data/processed/mini_test_set/labels/train/  # path to the location of partitions
```

Comment:
Actually, I think I still need to change the processed directory structure because:
1. It's not the structure recommended by [YOLO](https://docs.ultralytics.com/datasets/#steps-to-contribute-a-new-dataset) - `train` dir should be above `images` and `labels`
2. I believe we'll need to split 'detect' and 'pose' datasets in the 'processed' directory, as the labels will be different

Reply:
I agree. We shoudl rework our folder structure in data. My suggestion would be to drop raw & processed and go for the following folder structure:

data/
├── ultimate_keypoint_detect
│   ├── images
│   │   ├── train
│   │   └── val
│   ├── labels
│   │   ├── train
│   │   └── val
│   └── ultimate_keypoint_detect.yaml
└── ultimate_object_detect
    ├── images
    │   ├── train
    │   └── val
    ├── labels
    │   ├── train
    │   └── val
    └── ultimate_object_detect.yaml

In my opinion we don't need those intermediate results. Its just junk that floods our hard drives.

Reply:
correction: Maybe it is worth storing supervisely labels in a seperate folder for each dataset (labels_supervisely)

Reply:
The structure looks good, although I'd move `yaml` 2 levels up to the `ultimate-pipeline` folder. This way we are free to drop the whole `data` folder when we seem fit and start the pipeline from scratch. Also, then there's no need to whitespace yamls in `.gitignore'.

Regarding 'raw' and 'processed': a separation of layers is an important concept in Kedro and Cookiecutter. The main motivation of keeping data in respective layers immutable (e.g. in 'raw') is that you can change processing logic (e.g. label conversion, or train/val/test split) and then re-run certain stages of the pipeline. 
Therefore we should keep 'raw' and 'processed' separate.

To address disk space: we _could_ have an additional cleanup step at the end of the pipeline to drop raw data (not sure if it should run by default, but in any case, Kedro CLI allows us to control the nodes to execute).